SPM Readme Edits (#140)
Esri/arcgis-runtime-toolkit-ios: README.md

* [ArcGIS Runtime SDK for iOS](https://developers.arcgis.com/ios/) 100.10.0 (or higher)
* Xcode 12.0 (or higher)

The *ArcGIS Runtime Toolkit for iOS* has a *Target SDK* version of *13.0*, meaning that it can run on devices with *iOS 13.0* or newer.

## Instructions

### Swift Package Manager

 1. Open your project in Xcode
 2. Go to *File* > *Swift Packages* > *Add Package Dependency* option 
 3. Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL

>  4. Choose version 100.10.0 or a later version. Click Next. Only version 100.10.0 or newer supports Swift Package Manager.

Reviewer: ```suggestion
 4. Choose version 100.11.0 or a later version. Click Next. Only version 100.11.0 or newer supports Swift Package Manager.
```

Author: oh boy yes nice catch thank you.

## Instructions

### Swift Package Manager

> 

Author: I tested this in a forked repo and it works ok, I don't think this can be tested here until the SPM PR is merged because the package manifest must be in the root of the repo on the main branch.

 1. Open your project in Xcode
 2. Go to *File* > *Swift Packages* > *Add Package Dependency* option 
 3. Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL

>  4. Choose version 100.10.0 or a later version. Click Next. Only version 100.10.0 or higher supports Swift Package Manager.

Author: I tested this in a forked repo and it works ok, I don't think this can be tested here until the SPM PR is merged because the package manifest must be in the root of the repo on the main branch.

 New to cocoapods? Visit [cocoapods.org](https://cocoapods.org/)

### Manual

 1. Clone or download this repo
 2. Drag and Drop the `arcgis-runtime-toolkit-ios` folder into your project through the Xcode Project Navigator pane

>  3. Add the *ArcGISToolkit* library in your app, by adding it to the Frameworks, Libraries, and Embedded Content section of the General pane for your app target. The *ArcGISToolkit* library contains the *ArcGIS Runtime SDK for iOS* library, so you don't need to add that separately

Reviewer: ```suggestion
 3. Add the *ArcGISToolkit* library in your app, by adding it to the Frameworks, Libraries, and Embedded Content section of the General pane for your app target. The *ArcGISToolkit* library contains the *ArcGIS Runtime SDK for iOS* library, so you don't need to add that separately.
```

### Manual

 1. Clone or download this repo
 2. Drag and Drop the `arcgis-runtime-toolkit-ios` folder into your project through the Xcode Project Navigator pane

>  3. Add the *ArcGISToolkit* library in your app, by adding it to the Frameoworks, Libraries, and Embedded Content section of the General pane for your app target.

Reviewer: ```suggestion
 3. Add the *ArcGISToolkit* library in your app, by adding it to the Frameworks, Libraries, and Embedded Content section of the General pane for your app target.  The *ArcGISToolkit* library contains the *ArcGIS Runtime SDK for iOS* library, so you don't need to add that separately
```

Author: I just edited the suggestion to get rid of these double spaces "  The *ArcGISToolkit*"

>  4. Add `import ArcGISToolkit` in your source code and start using the toolkit components 

Reviewer: ```suggestion
 4. Add `import ArcGIS` and `import ArcGISToolkit` in your source code and start using the toolkit components 
```

Author: yep yep good catch
